CruzHacks/cruzhacks-2021-website
# Added Remaining Speakers

Problem & Solution 
=======
We did not have the complete lineup of speakers, so the workaround was having a `ComingSoon` component for the cards; however, we not have the final lineup ready and I've added the images and added the cards. 

Steps to Verify:
----------------
1. Checkout this branch and start a development environment 
2. Ensure that you see the speakers in alphabetical order based on Opening v Closing

Screenshots:
-----------------------
**Before**
<img width="1440" alt="Screen Shot 2021-01-09 at 5 14 04 PM" src="https://user-images.githubusercontent.com/40477553/104112012-1674dd00-529e-11eb-993b-dfa041aa7abc.png">

**After**
<img width="1440" alt="Screen Shot 2021-01-09 at 5 14 32 PM" src="https://user-images.githubusercontent.com/40477553/104112013-24c2f900-529e-11eb-8e58-03568d4e7967.png">
